pkg/operator/mon/mon.go

```diff
@@ -153,6 +141,8 @@ func (c *Cluster) CheckHealth() error {
 	return nil
 }
 
+// Don't call this function unless we have implemented monmap injection.
+// Otherwise updating the endpoints should not be supported.
 func (c *Cluster) updateConfigMapIfPodIPsChanged() error {
```

Comment:
no, i had left it here because it would be used when monmap injection is implemented. But it looks like that isn't going to be our approach now so i'll remove it.